docs/developers-clear-containers-install.md

```diff
@@ -1,5 +1,12 @@
 # Developers Clear Containers 3.0 Install
 
+* [Initial setup](#initial-setup)
```

Comment:
I actually used a hacky script that requires manual post-processing to work with both github (and `pandoc` for local testing) but I believe @sameo found a website that does it when you feed it an `.md` file. Wouldn't it be awesome if github supported auto-generating a TOC, like you get for free with rST ;(

Reply:
it would be nice if github handled it automatically (they do have their own md flavour - so should be able to add it ...) but, there are some tools around:
https://github.com/ekalinin/github-markdown-toc
https://github.com/jonschlinkert/markdown-toc

We could test and pick one for sure :-)

Reply:
yep - we should prolly discuss this for kata too at some point...